ui/packages/catalog-ui-search/src/main/webapp/js/Common.js

```diff
@@ -265,6 +265,15 @@ define([
                 this.wrapMapCoordinates(lon, [-180, 180]),
                 this.wrapMapCoordinates(lat, [-90, 90])
             ]);
+        },
+        getArrayFromValue: function(value) {
+          if (value === undefined || value === '' || value === null) {
+            return []
+          } else if (Array.isArray(value)) {
+            return value
+          } else {
+            return [value]
+          }
```

Comment:
This checks if it's an array, if it is an array already then it returns that array. Elsewise it makes sure that values which should evaluate as empty become empty arrays while scalar values are wrapped in an array.

Reply:
Will 0 ever be passed in? My understanding was that 0 would be of value '0' in dropdowns

Reply:
No idea but better safe than sorry.

Reply:
This function is in `Common`; it is not specific to dropdowns. `return value ? [value] : []` would not provide the expected functionality because `0` and `false` are expected to become `[0]` and `[false]`, respectively, while their falsy values would cause the ternary operation above to return `[]` for both.